src/julia/Float.jl

```diff
@@ -120,6 +120,14 @@ function Base.sqrt(a::AbstractFloat; check::Bool=true)
    return Base.sqrt(a)
 end
 
+function issquare(a::AbstractFloat)
+   return true
+end
+
+function issquare_with_sqrt(a::AbstractFloat)
+   return true, Base.sqrt(a)
+end
+
```

Comment:
I guess the remaining corner cases we can fix later if there are issues, but these two is glaring. `Base.sqrt` does not think that `Float64(-1)` is a square.

Reply:
Thanks. Those are indeed bugs on my part!

I'm attempting to get the other code cov cases covered, but it's not trivial to do locally. I thought I had them covered, but well, you can see that's not the case. Probably some silly errors on my part.

Reply:
Oh, I think I got them this time.

Reply:
Hopefully it is all ok now. Time to do Laurent and Puiseux series, which I'll do on a separate PR.